integration_tests/accessibility/accessibility.spec.ts

```diff
@@ -20,8 +20,8 @@ const urls = [
     url: `${routeUrls.cases}/123/contact-history?dateFrom-day=13&dateFrom-month=24&dateFrom-year=22&dateTo-day=14&dateTo-month=20&dateTo-year=22`,
   },
   // recommendation flow
-  { url: `${routeUrls.recommendations}/456/recall-type` },
-  { url: `${routeUrls.recommendations}/456/recall-type`, validationError: true },
+  // { url: `${routeUrls.recommendations}/456/recall-type` },
```

Comment:
You mean to comment this out?

Reply:
Yes, will be reintroduced when E2E working, see above